model/menu.js

```diff
@@ -5,7 +5,7 @@ const Schema = mongoose.Schema;
 
 const menuSchema = Schema({
     //TODO: menuSchema needs to have an array of pics for multiple page menus
-  // picId: { type: Mongoose.Schema.Types.ObjectId },
+  picURI: { type: String },
```

Comment:
This is ok for now, although we should think about how we'll move to multiple images.